### ISSUE TITLE
Node Pool Labels does nothing

### ISSUE THREAD

**Issue by regnaio** [vultr/govultr#306]
[Vultr API - Create NodePools](https://www.vultr.com/api/#tag/kubernetes/operation/create-nodepools)
> `labels` (`object`): Map of key/value pairs defining labels to automatically apply to all nodes in this nodepool. Labels will be applied to both new and existing nodes.

I added multiple key/value pairs for `labels` using `govultr`, but these labels are not applied to the nodes according to `kubectl get no --show-labels`

I'm confused what is the point of specifying `labels` if they are not applied to nodes. Also, these labels don't appear in the Vultr Web UI

Basically, the `labels` field does nothing

**Comment by optik-aper** [comment]
This is likely an issue in the API but I'll review the logic in govultr to be sure.

**Comment by optik-aper** [comment]
Ah, yes.  It appears that the create node pool doesn't process labels.  But that's not govultr's doing; the [create params](https://www.vultr.com/api/#tag/kubernetes/operation/create-kubernetes-cluster) don't include them in the API.

We'll have to address that there.  But you ought to be able to create and then update at this point.

**Comment by optik-aper** [comment]
I'll keep this open until we decide how to fix the create labels but I think it will be added soon.

**Comment by bcessa** [comment]
I believe this might be related to the Terraform provider not adding the labels to node pools provisioned with it. Is there any update on this issue? Thnx
